.gitattributes

```diff
@@ -0,0 +1,9 @@
+#
```

Comment:
Replace this file with
```
# Ensure all Java files use LF.
*.java eol=lf
```

Reply:
Fixed in [2a31fb6](https://github.com/ballerina-platform/module-ballerina-ldap/pull/45/commits/2a31fb6b1523983ad8d5dd04b3e57f10cd71427e)